### PR TITLE
Added benchmark for ignore3vl

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: tracks/latest.toml [hunk 1]
```diff
@@ -49,4 +49,6 @@ full = [
     "../specs/in_subquery.toml",
     "../specs/union.toml",
     "../specs/select/disabled_doc_values.toml"
```
Comment (mfussenegger) [inline tracks/latest.toml]: missing trailing comma?

File: specs/ignore3vl.toml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+[setup]
+statements = [
+    "create table t(a array(integer))",
+    "copy t from 's3://crate.benchmark.data/table_int_array.json.gz' with (compression = 'gzip')",
+    "refresh table t"
+]
+
+# All 250k rows match
+#[[queries]]
+#statement = "select count(*) from t where not 101 = any(a)"
```
Comment (mfussenegger) [inline specs/ignore3vl.toml]: Can we remove these?

**Comment by matriv** [comment]
Was thinking to do the rnd tests in another PR, but I can include here.

**Comment by matriv** [comment]
@mfussenegger Please check new commit and comments, Thx!

File: specs/ignore3vl.toml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+[setup]
+statements = [
+    "create table t(a array(integer))",
+    "copy t from 's3://crate.benchmark.data/table_int_array.json.gz' with (compression = 'gzip')",
```
[the remark this answers is not in the document]
Comment (mfussenegger) [reply specs/ignore3vl.toml]: No currently not - you could contribute it to cr8

File: specs/ignore3vl.toml (hunk 2 above)
Comment (mfussenegger) [reply specs/ignore3vl.toml]: Without comments it's hard to infer what they're for - looks like dead code. I'd remove them unless you'd like to chart the results of these queries as well, in which case I'd fully include them

**Comment by matriv** [comment]
@mfussenegger pushed fixup

**Pull request merged**